index.html

```diff
@@ -6,38 +6,67 @@
     <title>TripHouse</title>
     <link rel="stylesheet" href="styles/style.css" />
   </head>
-  <body>
+  <body class="body">
     <div class="advantages">
       <h2 class="title">What do we offer</h2>
 
       <hr class="underline" />
 
-      <div class="inline-block">
-        <div class="image-block">
-          <img src="./images/support.png" alt="headphones" />
+      <div class="container">
+        <div class="item">
+          <img class="image" src="./images/support.svg" alt="headphones" />
+          <p class="description">Support 24/7</p>
         </div>
-        <p class="description">Support 24/7</p>
-      </div>
 
-      <div class="inline-block">
-        <div class="image-block">
-          <img src="./images/communicate.png" alt="filter applied" />
+        <div class="item">
+          <img
+            class="image"
+            src="./images/communicate.svg"
+            alt="filter applied"
+          />
+          <p class="description">Communicate directly</p>
         </div>
-        <p class="description">Communicate directly</p>
-      </div>
 
-      <div class="inline-block">
-        <div class="image-block">
-          <img src="./images/book.png" alt="computer" />
+        <div class="item">
+          <img class="image" src="./images/book.svg" alt="computer" />
+          <p class="description">Book online</p>
+        </div>
+
+        <div class="item">
+          <img class="image" src="./images/reviews.svg" alt="cool" />
+          <p class="description">Real guest reviews</p>
         </div>
-        <p class="description">Book online</p>
       </div>
+    </div>
 
-      <div class="inline-block">
-        <div class="image-block">
-          <img src="./images/reviews.png" alt="cool" />
+    <div class="footer">
+      <div class="footer-container">
+        <div class="about">
+          <p>About</p>
```

Comment:
if you look carefuly in figma -you fid out that element have some spesial font-weight parameter, so you need class for that